### PR TITLE
Format hour/minute select boxes

### PR DESCRIPTION
**Pull request by serene**

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/forms.scss [hunk 1]
```diff
@@ -164,6 +164,14 @@ input[type="checkbox"]:checked + label{
   margin: 0 5%;
 }
 
+.hour, .minute{
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout_components/forms.scss]: These should go on separate lines, as per http://ama-layout.herokuapp.com/test2

**Comment by MathieuGilbert** [comment]
:hankey:

**Comment by serene** [comment]
:wave:

**Comment by kaytcampbell** [comment]
:art: :art: :art: :elephant: :elephant: :elephant:

**Pull request merged**